cmd/acraserver/listener.go

```diff
@@ -131,6 +208,54 @@ func (server *SServer) Start() {
 	}
 }
 
+func (server *SServer) Stop(socket *net.TCPListener) {
+	socket.SetDeadline(time.Now())
+}
+
+func (server *SServer) ListenerFD(socket *net.TCPListener) (uintptr, error) {
```

Comment:
What about to `ListenerFileDescriptor`? `uintptr` and `FD` not so obvious imho
Plus this method doesn't use any fields from `SServer` struct and better to implement it as function. Than we can use it in other places where will need to get descriptor from `TCPListener`
Plus we support not only `TCPListener`, `UnixListener` too (and connections). So what about to write something like that - https://play.golang.org/p/LuUqCB2YqyP ? Wrap it into functions `GetDescriptorFromListener[Connection]` and use where we need

Reply:
Refactored